modules/openapi-generator/src/main/java/org/openapitools/codegen/templating/MustacheEngineAdapter.java

```diff
@@ -64,8 +66,16 @@ public String compileTemplate(TemplatingExecutor executor, Map<String, Object> b
                 .withLoader(name -> findTemplate(executor, name))
                 .defaultValue("")
                 .compile(executor.getFullTemplateContents(templateFile));
+        StringWriter out = new StringWriter();
 
-        return tmpl.execute(bundle);
+        Object parent = bundle.get(CodegenConstants.CONFIG);
```

Comment:
can you please add comments explaining what this code block does and what exactly is CodegenConstants.CONFIG?

Reply:
Comments added
CONFIG renamed to MUSTACHE_PARENT_CONTEXT

Reply:
thanks. that's much better